src/styles/common.scss

```diff
@@ -16,7 +16,11 @@ body {
   margin: 0;
   color: $color-grey-text-1;
   font-family: 'Source Sans Pro', sans-serif;
-  overflow-y: hidden;
+  overflow: hidden;
   line-height: 1.5;
   font-size: $font-size-default;
 }
+
+.image-viz-hidden {
```

Comment:
Yeah. Replied about the HTML wrapper thing above. Many components like `react-rnd` or `react-draggable` used here can only be styled by classnames. Cannot make a wrapper over the `react-rnd` thing either because that will interfere with the drag and drop. Not sure how to fix this particular case.